scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_prep.sh
+#
+# This script processes the IMS input ascii/netcdf files onto the UFS model grid.
+# Inputs: IMS ascii/netcdf file, IMS index file (is model resolution specific) 
+#         and model restart files.
+# Outputs: file with i) IMS-derived snow cover fraction over land on UFS model 
+#         grid, and ii) snow depth derived form the IMS snow cover fraction, 
+#         using an inversion of the noah model snow depletion curve.
+#
+################################################################################
+#  Utilities
+RES=${CASE:1}
+
+#  Directories.
+export gdasapp_dir=${gdasapp_dir:-"${HOMEgfs}/sorc/gdas.cd/"}
```

Comment:
Why is `gdasapp_dir` needed?

Reply:
If the above `PYTHONPATH` issue is solved, this `gdasapp_dir` will be removed.